tests/test_analysis.py

```diff
@@ -91,3 +92,13 @@ def test_that_metrics_helper_works_for_data_with_the_same_label():
         warnings.filterwarnings('ignore', category=RuntimeWarning)
         evals = metrics_helper(same_human_scores, system_scores)
         assert_equal(evals.isnull().values.sum(), 1)
+
+
+def test_compute_pca_less_components_than_features():
+    # test pca when we have less components than features
+    df = pd.DataFrame({'a':range(100)})
+    for i in range(100):
+        df[i] = df['a']*i
+    (components, variance) = compute_pca(df, df.columns)
```

Comment:
Hm.. May be I should an actual test statement here, not just implicitly testing it does not crash...